Fully responive web

abraham-kwizera opened the pull request:
Hello Sir, 

This pull request contains codes regarding: 
1.  fully responsive: 
                - portfolio
                - blog page 
                - login page 
                - password restore page 
                - register page

2. update regarding stands up recommendations 

I will be glad to receive a review of my codes.

Abraham Kwizera.

abraham-kwizera committed:
Add hero section

abraham-kwizera committed:
Add about section

abraham-kwizera committed:
Add About, Testimonials and contacts sections

abraham-kwizera committed:
About ME: Update card with backend info

abraham-kwizera committed:
Update index with Login nav

abraham-kwizera committed:
create blog page, add header and footer section

abraham-kwizera committed:
On blog: Add latest blog wall

abraham-kwizera committed:
Add ancient post part, and subscribe part

abraham-kwizera committed:
create Login page

abraham-kwizera committed:
add password restore page

abraham-kwizera committed:
restore.hmtl: update email field

abraham-kwizera committed:
Add register page

abraham-kwizera committed:
Update Login button

abraham-kwizera committed:
Update Register button

abraham-kwizera committed:
Update Restore button

abraham-kwizera committed:
Update email, password fields

abraham-kwizera committed:
blog: Update email

abraham-kwizera committed:
Update subscription email field

abraham-kwizera committed:
update password field

abraham-kwizera committed:
revert Logo changes to ancient

abraham-kwizera committed:
update testimonial card by removing whitespaces, and reduce shadows

abraham-kwizera committed:
Highlight links to make them visible to user

abraham-kwizera committed:
add a fully responsive login page

abraham-kwizera committed:
Add a fully responsive password restore page

abraham-kwizera committed:
Add a fully responsive register page

abraham-kwizera committed:
add a fully responsiveness

abraham-kwizera committed:
Add responsiveness to blog page

abraham-kwizera committed:
add css doc

abraham-kwizera committed:
fix icon padding issue in media with maxi-width of 700px

abraham-kwizera committed:
Add firebase authentication for user registration

mansurissa commented on js/index.js:
```diff
@@ -1,3 +1,169 @@
 function toggleMobileMenu(menu) {
     menu.classList.toggle('open');
+}
```
Delete comented code

abraham-kwizera replied:
Thank you for your reviews, Sir,
I've already made changes regarding the reviews given in this file. 
Yours.

mansurissa commented on register.html:
```diff
@@ -56,41 +57,46 @@
-    <script src="./js/index.js"></script>
+            </>
+            <!-- </div> -->
+            <!-- </div> -->
```
Same here delete commented code

abraham-kwizera replied:
Thank you for your reviews, Sir,
I've already made changes regarding the reviews given in this file. allow me to know if changes are seen. 
Yours.

mansurissa reviewed (commented):
Delete comented code

mansurissa reviewed (changes requested):
Work on comments provided

abraham-kwizera committed:
Delete commented codes

abraham-kwizera committed:
Ft-backend integration (#7)

* Add firebase authentication for user registration

* Add firebase authentication for user login

* remove unneccessary comments

* Add logout button and js session

* Add add new post page

* Add admin messages page

* Add logo content in the header section

* Add logout button and logo in header

* Add js files and firebase connection

* add firebase files to .gitignore file

* Add integrate page with backend fetch api

* Integrate back end  with admin dashboard

* add image that appears when no profile picture

* Add sample images to be used

* update

* Remove commented codes

* Add alert and error alert layout designs

* Add alert message on contact form

* Add alert message on the form

* Update gitignore file

* Remove commented codes

* Remove un used Project link in navigation bar

* ch(remove testimonials setcion)
remove testimonial link in navigation bar
remove testimonials dummy designs and texts

wip

* ch(Update hero image layout to fit inplace)
Removed commented code in css
updated the width, height and margin of the layout

* Update additional info card discription

* ch(Remove commented code)
remove commented codes in style.css file

abraham-kwizera committed:
Add about section background

abraham-kwizera committed:
Merge branch 'main' into development
